content/zh-tw/zero-trust-architecture.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 零信任架構
+status: Completed
+category: 概念
+tags: ["安全性", "", ""]
+---
+
+## 是什麼 {#what-it-is}
```

Comment:
[MD022 - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md022.md)

Reply:
fixed